crochet-stitcher/src/index.ts

```diff
@@ -2,3 +2,5 @@ export { parse } from './stages/1-parser.js';
 export { link } from './stages/2-linker.js';
 export { place } from './stages/3-placer.js';
 export { elaborate } from './stages/4-elaborator.js';
+
+export * as models from './models/index.js';
```

Comment:
Hmm, I suppose this works too... My original idea was to just do `import {...} from 'crochet-stitcher/models'`, so re-exporting the models here wouldn't be necessary.

Reply:
(this also had editor issues and its own bugs before defining "." exports)

Reply:
> (this also had editor issues and its own bugs before defining "." exports)

It should work... mind elaborating?

Reply:
![image](https://github.com/user-attachments/assets/784b4b39-f50c-4775-ad06-3b4b3a900344)
![image](https://github.com/user-attachments/assets/1574989d-fc3f-4104-ba64-2dcb4c9e938c)